pages/stories.js

```diff
@@ -261,10 +261,9 @@ function Stories() {
                 ];
                 setPostComments(updatedPostComments);
 
-                setComments({...comments, [postId]: ''});
-                setOpenCommentInput(null);
+                setComments({...comments, [postId]: '' });
             } catch (error) {
-                console.error('Error adding comment:', error);
+                  console.error('Error adding comment:', error);
```

Comment:
Please remove the leading spaces which were added in front of console.error, thanks!

Reply:
Copy that!

Reply:
Looks good.